install/jaxws/bin/ts.jte

```diff
@@ -76,6 +76,11 @@ jaxws.lib.ri=${jaxws.home.ri}/modules
 endorsed.dirs=${jaxws.home}/modules/endorsed
 endorsed.dirs.ri=${jaxws.home.ri}/modules/endorsed
 
+###########################################################################
+# local.classes should contain any RI specific jars/classes
+###########################################################################
+local.classes=${ts.home}/glassfish5/glassfish/modules/jakartaee-api.jar${pathsep}${ts.home}/glassfish5/glassfish/modules/jakarta.xml.ws-api.jar${pathsep}${ts.home}/glassfish5/glassfish/modules/jakarta.xml.bind-api.jar
```

Comment:
JAX-WS uses `jaxws.classes` rather than local.classes. Assumingly, that should remain as such. Vendors supposedly have scripts that modify that property.

Reply:
ok